Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenStick.cs

```diff
@@ -430,6 +481,10 @@ public bool useIsolatedInputActions
         private List<RaycastResult> m_RaycastResults;
         [NonSerialized]
         private PointerEventData m_PointerEventData;
+        [NonSerialized]
+        private TouchControl m_TouchControl;
```

Comment:
Not really related to this fix, but more to the original design, this implies that a member would need to be added for any kind of pointer device right? Or is this particularly due to TouchControl not being a derivative of pointer in your opinion @bmalrat ?

Reply:
Normally we don't need it for each type, only for touch to identify which one was touched in the <Touchscreen>/touch*/press. I'm not aware of an other type that could aggregate multiple source into one binding.